src/main/content/_assets/js/guide-static.js

```diff
@@ -46,6 +46,28 @@
         $(".prereqs_list").html(prereq_html);
     });
 
+    // Read skills network json to see if we need to link to a cloud hosted guide equivalent.
+    $.getJSON( "../../guides/guides-common/cloud-hosted-guides.json", function(data) {
+        var guide_name = window.location.pathname.replace('.html','').replace('/guides/', '');
+        var host = window.location.hostname;
+        var skills_network_url = host === "openliberty.io" ? data.skillNetworkUrl : data.stagingSkillNetworkUrl;
+        skills_network_url = skills_network_url.replace('{projectid}', guide_name);
+        if(data.guides.indexOf(guide_name) > -1){
+            $('.skills_network_description').text(data.tooltipText);
+            var skills_network_button = $('<a class="skills_network_button" target="_blank"></a>');
```

Comment:
You should rel="noopener noreferrer" for all target of _blank

Reply:
I think we have build code that adds that automatically for us?  @steven1046 can confirm

Reply:
confirmed that it's there on the staging site:
![image](https://user-images.githubusercontent.com/3076261/104358361-f3cc0980-54d3-11eb-8be9-2e6dbcb4dec5.png)